src/main/scala/piconot/PicoFSM.scala

```diff
@@ -0,0 +1,98 @@
+package piconot
+
+import scalafx.application.JFXApp
+import picolib.maze.Maze
+import picolib.semantics
+import scala.collection.mutable.MutableList
+import scala.language.experimental.macros
+
+trait PicoBot extends JFXApp {
```

Comment:
Interesting choice to build everything together. Really simplified Empty/RightHand.